guides/contribution-guide.md

```diff
@@ -1,13 +1,25 @@
 # Guidelines for Contribution
-###### Last updated February 20, 2018
+###### Last updated March 10, 2018
 
 :::
-In order for a contribution to be considered, the following conditions must be met:
+##### All may contribute
+We would love for you to contribute to Cashmere and help make it ever better! As a contributor, you will be part of a community making this project better. To get started, just follow the [getting started guide](http://cashmere.healthcatalyst.net/guides/getting-started)

```

Comment:
Not sure what is happening, but when I click the Getting Started guide link in the ng serve build, it goes to the "right" link but the content is off.
![2018-03-17 09_40_07-cashmere](https://user-images.githubusercontent.com/15255009/37557242-4846e66e-29c7-11e8-9d42-6f14415385be.png)
 Maybe that is something wrong on my end.